sdk/eventgrid/azure-eventgrid/azure/eventgrid/_publisher_client.py

```diff
@@ -124,8 +124,13 @@ def _policies(credential, **kwargs):
         return policies
 
     @distributed_trace
-    def send(self, events, **kwargs):
-        # type: (SendType, Any) -> None
+    def send(
+        self,
+        events: SendType,
+        *,
+        channel_name: str = None,
```

Comment:
Isn't this a contradiction? If None is a value that is acceptable (acceptable being defined as doesn't raise), then I guess it should be `Optional[str]`? Does this code pass mypy? Do we have mypy enabled on EventGrid?

Reply:
it passes mypy - and we have it enabled - this is the precedence we have been using for keyword only (we don't tag it "optional")